services/userService.js

```diff
@@ -0,0 +1,110 @@
+const jwt = require('jsonwebtoken');
+const requestPromise = require('request-promise');
+const userDao = require('../models/userDao');
+
+const signIn = async ( code ) => {
+  const apiKey  = process.env.API_KEY;
+
+  const getKakaoTokenOption = {
+    uri: "https://kauth.kakao.com/oauth/token",
+    method: "POST",
+    form: {
+      grant_type: "authorization_code",
+      client_id: apiKey,
+      redirect_uri: "http://127.0.0.1:3000/user/kakao/signin",
+      code: code
+    },
+    headers: {
+        "content-type" : "application/x-www-form-urlencoded"
+    },
+    json: true
+  }
+
+  const getKakaoToken = await requestPromise(getKakaoTokenOption);
+
+  const getKakaoUserInfo = {
```

Comment:
위에서 인가코드를 받아올 때의 변수명을 고려하여 네이밍을 해주면 좋을 것 같습니다!

```suggestion
  const getKakaoUserOption = {
```